### PR TITLE
[SLDEV-13650] Add Cypress integration example

### PR DESCRIPTION
**Pull request by kristijanstefanoski**

### CODE REVIEW

File: cypress-browser-agent-otel/cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
+// This example commands.js shows you how to
```
Comment (alonweiss-sl) [inline cypress-browser-agent-otel/cypress/support/commands.js]: Is this file required?

File: cypress-browser-agent-otel/cypress/e2e/spec.cy.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      const customEvent = new CustomEvent("delete:baggage");
+      win.dispatchEvent(customEvent);
+    });
+  });
+
+  after((done) => {
+    // End the current test session after the running suite using Sealights Public API
+    SLService.endTestSession(testSession).finally(() => {
+      cy.window().then(async (win) => {
+        win.$SealightsAgent.sendAllFootprints().finally(done);
```
Comment (alonweiss-sl) [inline cypress-browser-agent-otel/cypress/e2e/spec.cy.js:63]: I'm very curious how this code executes. My understanding is that everything inside the (win)=>{...} code happens inside the browser, but it calls done() which is defined on line 59 and runs inside the cypress context. 

Did you verify this is actually executing? For example, if sendAllFootprints() would now sleep for 10 seconds (just for testing), will this cause the test to wait as well?

**Review by alonweiss-sl — CHANGES_REQUESTED** [review]
See 2 minor comments

File: cypress-browser-agent-otel/cypress/support/commands.js (hunk 1 above)
Comment (kristijanstefanoski) [reply cypress-browser-agent-otel/cypress/support/commands.js]: Not at the moment, just boilerplate file - will check if it can be removed

File: cypress-browser-agent-otel/cypress/e2e/spec.cy.js (hunk 2 above)
Comment (kristijanstefanoski) [reply cypress-browser-agent-otel/cypress/e2e/spec.cy.js:63]: It's supposed to work like that but I will test this right now before confirming if it actually waits that long or might have timeout

Comment (kristijanstefanoski) [reply cypress-browser-agent-otel/cypress/e2e/spec.cy.js:63]: Verified with following code for future references:

```
after(function (done) {
  this.timeout(20000);
  // End the current test session after the running suite using Sealights Public API
  SLService.endTestSession(testSession).finally(() => {
    cy.window().then(async (win) => {
      win.$SealightsAgent.sendAllFootprints = () => {
        return new Promise((resolve) => {
          setTimeout(() => {
            resolve();
          }, 10000);
        });
      };
      win.$SealightsAgent.sendAllFootprints().finally(done);
    });
  });
});
```

**Review by alonweiss-sl — APPROVED** [review]
